webapp/tests/utils/formatting_at_mentions.test.jsx

```diff
@@ -51,16 +51,16 @@ describe('TextFormatting.AtMentions', function() {
     });
 
     it('Implied at mentions', function() {
-        // PLT-4454 Assume users exist for things that look like at mentions until we support the new mention syntax
+        // implied at mention is presented as is and non-clickable
         assert.equal(
             TextFormatting.autolinkAtMentions('@user', new Map(), {}),
-            '$MM_ATMENTION0',
+            '@user',
             'should imply user exists and replace mention with token'
```

Comment:
Also, can you either update or remove these messages about what the expected behaviour is? I'm 0/5 on having them, but if they're here, they should be correct about the expected behaviour.